bootstrap-salt.ps1

```diff
@@ -40,14 +40,14 @@ is dynamically determined by the script.
 Bootstrap GitHub Project (script home) - https://github.com/saltstack/salt-windows-bootstrap
 Original Vagrant Provisioner Project -https://github.com/saltstack/salty-vagrant
 Vagrant Project (utilizes this script) - https://github.com/mitchellh/vagrant
-SaltStack Download Location - http://docs.saltstack.com/downloads/
+SaltStack Download Location - https://repo.saltstack.com/windows/
 #>
 [CmdletBinding()]
 Param(
   [Parameter(Mandatory=$false,ValueFromPipeline=$true)]
   # Doesn't support versions prior to "YYYY.M.R-B" 
   [ValidatePattern('^(201[0-9]\.[0-9]\.[0-9](\-\d{1})?)$')]
-  [string]$version = "2015.5.2",
+  [string]$version = "2015.8.3",
```

Comment:
@jfindlay I apologize, I mistakenly thought that here we got the changes made by @kenerwin88.
Now I clearly see there are no commits authored by him. Sorry for the inconvenience.

I guess all we can do here is just to bump Salt version...